src/Diagnostics.RuntimeHost/Controllers/DevOpsController.cs

```diff
@@ -77,7 +77,7 @@ public async Task<IActionResult> GetFileContentAsync(string filePathInRepo, stri
         [HttpGet(UriElements.DevOpsGetBranches)]
         public async Task<IActionResult> GetBranchesAsync(string resourceUri)
         {
-            object response = await _devOpsClient.GetBranchesAsync(resourceUri, this.HttpContext.Request.Headers[RequestIdHeaderName]);
+            List<(string, bool)> response = await _devOpsClient.GetBranchesAsync(resourceUri, this.HttpContext.Request.Headers[RequestIdHeaderName]);
```

Comment:
I do see other methods in this controller that return Object type (MakePullRequest, GetFileContent, PushChanges). Are those working as expected?

Reply:
GetFileContent and PushChanges are working as expected. I made changes for MakePullRequest in an update to the PR